test/PowerNotificationsTests/PowerNotificationsTests.vcxproj

```diff
@@ -91,12 +91,13 @@
       <ConformanceMode>true</ConformanceMode>
       <PrecompiledHeader>Use</PrecompiledHeader>
       <PrecompiledHeaderFile>pch.h</PrecompiledHeaderFile>
-      <AdditionalIncludeDirectories>$(OutDir)\..\ProjectReunion_DLL</AdditionalIncludeDirectories>
+      <AdditionalIncludeDirectories>$(ProjectDir)..\..\packages\Microsoft.Taef.10.58.210222006-develop\build\Include;$(OutDir)\..\ProjectReunion_DLL</AdditionalIncludeDirectories>
```

Comment:
This is fine for now, but isn't ideal (as I'm sure you're aware, since this issue is presumably the reason you were using the other NuGet feed for TAEF). Having the package version number copied around everywhere is obnoxious and adds friction for whenever we need to update the package version.

I believe that packages\Microsoft.Taef.Redist.Wlk.10.52.200221001\build\Taef.Redist.Wlk.targets is _supposed_ to auto-inject the appropriate properties for consuming TAEF into projects that consume the NuGet package. I'm not 100% sure why this doesn't work; possibly just because the it's supposed to be named **Microsoft**.Taef.Redist.Wlk.targets to match the NuGet package name.
IMO, we should follow up separately with Reunion ES and TAEF folks and get this fixed properly. The best option would be to fix the NuGet package. Barring that, we might still consider having a centralized .props file tracking the project's TAEF version (assuming we want to standardize which TAEF version is used; I guess that's non-essential, since all of the test binaries build independently of each other).

Reply:
Excellent point. I will follow up with them.